### PR TITLE
Show errors in scheduled messages

### LINKED ISSUE #170: Display validation errors in scheduled messages
Related to #147

### PR DESCRIPTION
**Pull request by aeatencio**
Resolves #170

### CODE REVIEW

File: spec/lib/backend_error_spec.rb [hunk 1]
```diff
@@ -483,127 +483,310 @@
     end
 
     it "returns empty for scheduled messages" do
```
Comment (nekron) [inline spec/lib/backend_error_spec.rb]: empty?

File: spec/lib/backend_error_spec.rb [hunk 2]
```diff
@@ -483,127 +483,310 @@
...
+              }
+            }]
+          }]
+        }
+      }]
+      errors_out = [{:message=>"required", :path=>["skills/0", "messages"]}]
+      expect(BackendError.parse_errors(errors_in)).to eq(errors_out)
+    end
+
+    it "returns empty for scheduled messages when fixed_time selected" do
```
Comment (nekron) [inline spec/lib/backend_error_spec.rb]: empty?

File: spec/lib/backend_error_spec.rb (hunk 1 above)
Comment (aeatencio) [reply spec/lib/backend_error_spec.rb]: Sorry, @nekron, I've forgotten to rename this test. It returned empty before these changes. I'll rename it in the next pull request. Thanks!

File: spec/lib/backend_error_spec.rb (hunk 2 above)
Comment (aeatencio) [reply spec/lib/backend_error_spec.rb]: Same here, inherited from the other.

**Pull request merged**